c_plus_plus/src/view/views/recording_view.cpp

```diff
@@ -6,9 +6,65 @@ namespace View
 
 RecordingView::RecordingView(QWidget *parent)
     : AbstractView("Recording", parent)
-    , ui(new Ui::RecordingView)
+    , mUi(new Ui::RecordingView)
 {
-    ui->setupUi(this);
+    mUi->setupUi(this);
+
+    mCameraViewfinder = new QCameraViewfinder(this);
```

Comment:
Call `, mCamera(new QCamera(getCameraInfo()))` after the initialization of mUi over calling it in the constructor body